test/IRGen/autorelease.sil

```diff
@@ -36,6 +37,10 @@ bb0(%0 : @owned $C?):
 // arm64:      [[T0:%.*]] = tail call i64 bitcast ([[OBJC]]* ([[OBJC]]*)* @objc_autoreleaseReturnValue to i64 (i64)*)(i64 %0)
 // arm64-NEXT: ret i64 [[T0]]
 
+// aarch64:    define{{( dllexport| protected)?}} swiftcc i64 @foo(i64) {{.*}} {
+// aarch64:      [[T0:%.*]] = tail call i64 bitcast ([[OBJC]]* ([[OBJC]]*)* @objc_autoreleaseReturnValue to i64 (i64)*)(i64 %0)
+// aarch64-NEXT: ret i64 [[T0]]
+
```

Comment:
I think that we should rewrite the `aarch64` target in `lit.cfg` to `arm64`.

Reply:
I wondered why that wasn’t the case already, but there was a lot of commit doing this kind duplication, and I wanted to follow the existing patterns. I can look into what your proposal will involve.

Reply:
So I was exploring this idea, but sadly it might not be possible.

There’s some tests that have different expectations for `arm64` and `aarch64`. One of those test is https://github.com/apple/swift/blob/master/test/IRGen/c_layout.sil#L224-L238 where the `arm64` checks include some `signext` and `zeroext` which fail when added to the `aarch64`.

Another problem I found is that the prefix comes from the substitution `%target-cpu`, but that substitution is used in a couple more places than as a prefix, the solution for that is easy, but not very nice.

With those two problems, I think it is better to not try to rewrite `aarch64` to `arm64`, and keep duplicating the expectations.